confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -942,6 +980,42 @@ private void readPageObject(XMLStreamReader xmlReader)
         }
     }
 
+    private void readBlogPostObject(XMLStreamReader xmlReader)
+        throws XMLStreamException, ConfigurationException, FilterException
+    {
+        ConfluenceProperties properties = new ConfluenceProperties();
+
+        properties.setProperty(KEY_PAGE_BLOGPOST, true);
+
+        long pageId = readObjectProperties(xmlReader, properties);
+
+        savePageProperties(properties, pageId);
+
+        Long spaceId = properties.getLong("space", null);
+
+        // Register blog post page
+        Long originalVersion = (Long) properties.getProperty("originalVersion");
```

Comment:
You should keep the comment of the code you copied from readPageObject which explain why this originalVersion manipulation is about.